NEWS.md

```diff
@@ -1,251 +0,0 @@
-## DataFrames v0.6.11 Release Notes
```

Comment:
Hmmm, looks like updating the descriptions in release notes will make a new release and attobot will submit a PR to METADATA.jl :(